CHANGELOG.md

```diff
@@ -29,6 +29,7 @@
 * Remove accessible format request pilot support ([PR #2826](https://github.com/alphagov/govuk_publishing_components/pull/2826))
 * GTM analytics add page views ([PR #2814](https://github.com/alphagov/govuk_publishing_components/pull/2814))
 * Add id attribute to error alert component ([PR #2825](https://github.com/alphagov/govuk_publishing_components/pull/2825))
+* Add js search to table component ([PR #2803](https://github.com/alphagov/govuk_publishing_components/pull/2803))
```

Comment:
You probably want to move this to under the `Unreleased` header

Reply:
OK, have moved this.